1-Authentication/1-sign-in/App/authPopup.js

```diff
@@ -84,10 +84,12 @@ function signOut() {
 
     // Choose which account to logout from by passing a username.
     const logoutRequest = {
-        account: myMSALObj.getAccountByUsername(username)
+        account: myMSALObj.getAccountByUsername(username),
+        mainWindowRedirectUri: 'http://localhost:3000/signout',
+        redirectUri: 'http://localhost:3000/redirect.html',
```

Comment:
```suggestion
        redirectUri: '/redirect', // e.g. http://localhost:3000/redirect
```

Reply:
Here and everwhere we can get rid of `.html` extension in redirect uri (we had to do this in React to make it work with react router)